site/content/docs/developing/overrides/yamlpatch.md

```diff
@@ -0,0 +1,123 @@
+# YAML Patch Overrides
+
+!!! Attention
+
+    :warning: Overriding CloudFormation templates is an advanced feature which can cause your stacks to not deploy successfully. 
+    Please use with caution!
+
+Copilot generates CloudFormation templates using configuration specified in the [manifest](../../manifest/overview.en.md). 
+However, not all CloudFormation properties are configurable in the manifest. 
+For example, you might want to configure the [`Ulimits`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-taskdefinition-ulimit.html) 
+for your workload container, but the property is not exposed in manifests.
+
+Overrides with `yamlpatch` or `cdk` allows you to add, delete, or replace _any_ property or resource in a CloudFormation template.
+
+## When should I use YAML Patch over CDK overrides?
+
+Both options are a "break the glass" mechanism to access and configure functionality that is not surfaced by Copilot [manifests](../../manifest/overview.en.md).  
+
+We recommend using YAML patch over the [AWS Cloud Development Kit (CDK) overrides](./cdk.md) if 1) you do not want to have a dependency
+on any other tooling and framework (such as [Node.js](https://nodejs.org) and the [CDK](https://docs.aws.amazon.com/cdk/v2/guide/home.html)), 
+or 2) you have to write only a handful modifications. 
+
+## How to get started
```

Comment:
it seems like everything above is very similar to cdk. maybe consider making a homepage for `override` with general discussion and `how to get started`, `examples` etc. in each page, so that users will clearly know they are two different approaches for the same problem

Reply:
oohh, i'm happy to use `include` so that we can have the duplicate blocks only in one location.

But I'm thinking of keeping the text in both pages because I don't want users to navigate between pages too much. Does that sound good to you too boss?